lib/analytics/course_edits_csv_builder.rb

```diff
@@ -37,8 +38,13 @@ def row(revision)
     row << revision.article.full_title
     row << revision.url
     row << revision.user.username
-    row << revision.characters
+    add_character_references(revision, row)
```

Comment:
I think this is clearer as keeping one line per piece of data. Doing just these two as their own method makes it harder to follow what's going on, and those two items are any more closely related than some of the other items.

Reply:
Rubocop was giving warning on Assignment Branch Assignment
So had to make a function and add it

Reply:
I think disabling that rubocop rule for the method is a better option for cases like this.

The idea behind the rule is to avoid methods that have too much complexity and are hard to understand, but in cases like this where we have a really clear and repetitive pattern — add something to a row, once for each column — it's more clear and simple to break the rule than try to work around it.

Reply:
Guess, it was giving that warning when was calculating it using features and features_previous and not by calling function